recipe/test.c

```diff
@@ -0,0 +1,8 @@
+#include <cublas_v2.h>
+
+int main() {
+    cublasStatus_t stat;
+    cublasHandle_t handle;
+    stat = cublasCreate(&handle);
+    return stat != CUBLAS_STATUS_SUCCESS;
```

Comment:
Q: Do we need any of these? Removing them can still test the same header dependency, but without the need to link to cuBLAS.
```suggestion
```

Reply:
Aha, that's why I see how some other recipes having the empty main() function. This sample code was suggested by @bdice and @robertmaynard in https://github.com/orgs/NVIDIA/projects/46/views/1?pane=issue&itemId=71981175 . Do you guys have a specific reason for wanting to link against cuBLAS lib ?

Reply:
We are thinking it would be useful to link against `libcublas` as it can be a good way to check that we are able to resolve symbols as part of the test